indexing-solr/src/test/resources/datastream/techmd_mp4.xml

```diff
@@ -0,0 +1,91 @@
+            <premis3:compositionLevel>0</premis3:compositionLevel>
+            <premis3:format>
+                <premis3:formatDesignation>
+                    <premis3:formatName>MPEG-4</premis3:formatName>
+                </premis3:formatDesignation>
+            </premis3:format>
+            <premis3:size>5298788</premis3:size>
+            <premis3:objectCharacteristicsExtension>
+                <fits xmlns="http://hul.harvard.edu/ois/xml/ns/fits/fits_output" xsi:schemaLocation="http://hul.harvard.edu/ois/xml/ns/fits/fits_output http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd" version="1.6.0" timestamp="7/9/24, 4:44 PM">
+                    <identification status="CONFLICT">
+                        <identity format="MPEG-4" mimetype="video/mp4" toolname="FITS" toolversion="1.6.0">
+                            <tool toolname="MediaInfo" toolversion="23.09"/>
+                            <tool toolname="Droid" toolversion="6.5.2"/>
+                            <externalIdentifier toolname="Droid" toolversion="6.5.2" type="puid">fmt/199</externalIdentifier>
+                        </identity>
+                        <identity format="symbolic link to `/opt/data/deposits/f0e63a5e-fac1-4120-bc4d-7cd119b3a3e4/data/_82101931-f10c-4c8f-a979-619da56037ed'" mimetype="inode/symlink" toolname="FITS" toolversion="1.6.0">
+                            <tool toolname="file utility" toolversion="5.11"/>
+                        </identity>
+                    </identification>
+                    <fileinfo>
+                        <filepath toolname="OIS File Information" toolversion="1.0" status="SINGLE_RESULT">/path/to/video.mp4</filepath>
+                        <filename toolname="OIS File Information" toolversion="1.0" status="SINGLE_RESULT">002.mp4</filename>
+                        <size toolname="MediaInfo" toolversion="23.09">5298788</size>
+                        <md5checksum toolname="OIS File Information" toolversion="1.0" status="SINGLE_RESULT">7e6c03925cc3782016e57eb6dc163f66</md5checksum>
+                        <fslastmodified toolname="OIS File Information" toolversion="1.0" status="SINGLE_RESULT">1720543039916</fslastmodified>
+                    </fileinfo>
+                    <filestatus/>
+                    <metadata>
+                        <video>
+                            <location toolname="MediaInfo" toolversion="23.09" status="SINGLE_RESULT">/path/to/object</location>
+                            <mimeType toolname="MediaInfo" toolversion="23.09" status="SINGLE_RESULT">video/quicktime</mimeType>
+                            <format toolname="MediaInfo" toolversion="23.09" status="SINGLE_RESULT">MPEG-4</format>
+                            <formatProfile toolname="MediaInfo" toolversion="23.09" status="SINGLE_RESULT">Base Media</formatProfile>
+                            <duration toolname="MediaInfo" toolversion="23.09" status="SINGLE_RESULT">86616</duration>
```

Comment:
It seems like FITS has already settled on a duration when video and audio disagree, maybe we can use this value instead of pulling from the tracks?